src/test/java/seedu/address/logic/commands/FindFreeTimeCommandTest.java

```diff
@@ -0,0 +1,68 @@
+package seedu.address.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.TypicalEvents.getTypicalSochedule;
+
+import java.util.ArrayList;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.common.Date;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DeleteTaskCommand}.
+ */
+public class FindFreeTimeCommandTest {
+
+    private Model model = new ModelManager(getTypicalSochedule(), new UserPrefs());
+
+    @Test
+    public void execute_validDateNoFreeTime_success() {
+        Date exampleDate = new Date("2021-03-16");
```

Comment:
The exampleDate can consider to be put in CommandTestUtil as a constant, otherwise, it looks like magic number.